syncd/FlexCounter.cpp

```diff
@@ -2122,6 +2224,19 @@ void FlexCounter::addCounter(
 
             setSwitchDebugCounterList(vid, rid, switchCounterIds);
         }
+        else if (objectType == SAI_OBJECT_TYPE_MACSEC_SA && field == MACSEC_SA_ATTR_ID_LIST)
+        {
+            std::vector<sai_macsec_sa_attr_t> MACsecSAIds;
```

Comment:
don't prefix local variables with capital letters

Reply:
Done, please review it.